src/Look/InfoBoxLook.cpp

```diff
@@ -73,10 +73,11 @@ InfoBoxLook::Initialise(bool _inverse, bool use_colors,
 }
 
 void
-InfoBoxLook::ReinitialiseLayout(unsigned width)
+InfoBoxLook::ReinitialiseLayout(unsigned width, unsigned scale_title_font)
 {
   FontDescription title_font_d(8);
-  AutoSizeFont(title_font_d, width, _T("123456789012345"));
+  unsigned scaled_width = (width * scale_title_font) / 100U;
```

Comment:
I don't understand this formula. What is this supposed to do?
And if fonts are auto-sized, how will this change affect the font sizing algorithm?

Reply:
AutoSizeFont picks the font such that the TCHAR string fits into the given width.
scaled_width gives it more or less space for the same string. See next line 80.

Reply:
I still don't understand. Why would you want to give it more space than is really available on the screen?